scripts/build_demand.py

```diff
@@ -0,0 +1,316 @@
+import os
+from _helpers import configure_logging, sets_path_to_root
+
+
+import geopandas as gpd
+import pandas as pd
+import fiona
+import rasterio
+import rasterio.mask
+import georasters as gr
+import pandas as pd
+import requests
+import shutil
+import logging
+import glob
+
+
+from _helpers import (
+    configure_logging,
+    sets_path_to_root,
+    three_2_two_digits_country,
+    two_2_three_digits_country,
+    two_digits_2_name_country,
+    get_country,
+)
+
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
+
+"""
+Estimates the population and the electric load of the microgrid.
+
+Relevant Settings
+-----------------
+.. code:: yaml
+
+    microgrids_list:
+        Location:
+            Centre:
+                lon:
+                lat:
+            Sides:
+                Deltalon:
+                Deltalat:
+        micA: 
+            name:
+
+    load:
+        scaling_factor:
+
+Inputs
+------
+- ``data/sample_profile.csv``: a load profile, which will be scaled through a scaling_factor to obtain the per person load
+
+Outputs
+-------
+- ``data/Worldpop/{country}_ppp_{year}_constrained.tif": a tif file of the population of the selected country,
+- ``microgrid_shape="resources/shapes/microgrid_shape.geojson": a geojson file of the shape of the microgrid,
```

Comment:
Cool draft description :)

To be a docstring, it shall be moved even before the imports.
So this entire block : """ ... """ shall be moved upwards at line 1 basically.

Moreover, in the outputs the name of the options shall be removed, no need for them

Reply:
Done.